src/demo/models/pond.js

```diff
@@ -40,7 +40,7 @@ const arrangeFish = fishes => {
     );
     const y = randomInt(
       0,
-      constants.canvasHeight - constants.fishCanvasHeight / 2
+      constants.canvasHeight - constants.fishCanvasHeight / 2 - 160
```

Comment:
what's the purpose of this change?

Reply:
Good question.  It stops fish showing behind the UI.  I updated with a variable and an explanation.